micro-rdk-server/esp32/ota_8mb_partitions.csv

```diff
@@ -1,7 +1,8 @@
 # Name,   Type, SubType, Offset,  Size, Flags
 # Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
-nvs,	       data,	nvs,	  0x9000,	0x6000,
-otadata,       data,	ota,	  0xF000,	0x2000,
-phy_init,      data,	phy, 	  0x11000, 	0x1000,
-ota_0,	       0,	ota_0,	  ,		0x377000,
-ota_1,	       0,	ota_1,	  ,		0x377000,
+nvs,         data,  nvs,      0x9000,   0x6000,
+otadata,     data,  ota,      0xF000,   0x2000,
+phy_init,    data,  phy,      0x11000,  0x1000,
+ota_0,       app,   ota_0,    0x20000,  3968K,
+ota_1,       app,   ota_1,    0x400000, 3968K,
```

Comment:
I also think we should be consistent about whether we use hex or K/M for sizes. I sort of like keeping the offsets in hex, I think. They don't change that often, and things get logged in that format. But sizes are nice to be able to read quickly in human units. Is it weird to have offsets in hex and sizes in K/M?

Reply:
I see your point. I like the offset in hex since it's easy to compare with address-space output in logs (I reference it a lot in ota).

For `otadata` at least, there is benefit in having it stay in hex because those are hard requirements according to the [docs](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/system/ota.html#ota-data-partition) (shown in hex) and would be easy to see if it's misconfigured.

I'm looking now if phy_init has similar requirements, but haven't come across anything yet.

Reply:
we could also have all sizes in K/M and have a trailing comment for the hex on otadata